pages/user.js

```diff
@@ -50,13 +79,18 @@ export default function TutorialsPage({ userTutorials, userName }) {
         <Message>You haven&apos;t created any tutorials yet</Message>
```

Comment:
Die Darstellung im Satz kommt mir komisch vor, ist aber sicherlich so vorgesehen?

Reply:
ESLint suggests this notation instead of an apostrophe.